packages/core/nuxt-theme-module/theme/components/MyAccount/ProfileUpdateForm.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <b>Please implement vendor specific ProfileUpdateForm component in 'components/MyAccount' directory</b>
+  <b>{{ $t('Please implement vendor specific ProfileUpdateForm') }}</b>
```

Comment:
I think we can omit prompts like this since they're just for developers, I'll mark them in comments with 🥥